features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/callback/RecorderWindowCallback.kt

```diff
@@ -54,16 +56,24 @@ internal class RecorderWindowCallback(
                 copy.recycle()
             }
         } else {
-            // TODO: RUMM-2397 Add the proper logs here once the sdkLogger will be added
-            // sdkLogger.errorWithTelemetry("Received MotionEvent=null")
+            internalLogger.log(
+                InternalLogger.Level.ERROR,
+                InternalLogger.Target.TELEMETRY,
```

Comment:
I feel that because we don't have any control on how we're being called by the Window, there's nothing that we could do even if we go a high volume of events telling us that MotionEvent are null. Unless we dug into the AOSP source code and know when such a situation can occur, I think this should just be a `MAINTAINER`  log

Reply:
yeah you are right, I was not sure of this one either. I can make it a `MAINTAINER` one only for visibility in debug.